lib/galaxy/tools/deps/conda_util.py

```diff
@@ -309,7 +324,22 @@ def is_target_available(conda_target, conda_context=None):
 
 def is_conda_target_installed(conda_target, conda_context=None):
     conda_context = _ensure_conda_context(conda_context)
-    return conda_context.has_env(conda_target.install_environment)
+    # fail by default
+    success = False
+    if conda_context.has_env(conda_target.install_environment):
+        # because export_list directs output to a file we
+        # need to make a temporary file, not use StringIO
+        f, package_list_file = tempfile.mkstemp(suffix='.env_packages')
+        os.close(f)
+        conda_context.export_list(conda_target.install_environment, package_list_file)
+        search_pattern = conda_target.package_specifier + '='
```

Comment:
@pvanheus 
This is a bit too restrictive for my taste, since the tool maintainer specifying the version should include the ("=")  if deemed necessary ... and this is in contradiction with what list_dependencies() produces :/.

Any chance you remember which package needed this more verbose checking ?